.github/workflows/uploadbuildtomuuktest.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Upload to Muuktest
+
+on:
+  push:
+    branches:
+      - release-candidate
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Xcode
+      run: sudo xcode-select --switch /Applications/Xcode.app
+
+    - name: Build the app
+      run: xcodebuild -scheme "SPM Example" -archivePath ${{ github.workspace }}/build/SPMExample.xcarchive archive
+
+    - name: Export the app
+      run: |
+        xcodebuild -exportArchive \
+        -archivePath ${{ github.workspace }}/build/SPMExample.xcarchive \
+        -exportPath ${{ github.workspace }}/build \
+        -exportOptionsPlist <(echo '
+        {
+          "method": "app-store",
+          "teamID": "WTQ4Q66FY6",
+          "provisioningProfiles": {
+            "io.portalhq.ios.SPM-Example": "io.portalhq.ios.SPM-Example1"
+          },
+          "signingStyle": "automatic",
+          "compileBitcode": true,
+          "thinning": "<none>"
+        }')
+
+    - name: Rename exported IPA file
+      run: mv ${{ github.workspace }}/build/SPM\ Example.ipa ${{ github.workspace }}/build/SPMExample.ipa
+
+    - name: Get upload URL
+      id: get_upload_url
+      run: |
+        response=$(curl -H --location 'https://11mi38nzh1.execute-api.us-east-2.amazonaws.com/v1/uploadFile' \
+          --header 'x-api-key: eLiCJrCf2R4uHMNpKAOcC1aTQkdFIy006CeWhFDj' \
```

Comment:
This should be moved to an environment variable, so we're not committing API Keys to git.

Reply:
Updated code to use env variable for api-key along with other stuff.